### PR TITLE
dev-vcs/git-cola: add 4.9.0

### PR DESCRIPTION
**Pull request by stkw0**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: dev-vcs/git-cola/git-cola-4.9.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+SLOT="0"
+KEYWORDS="amd64 x86"
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/numpy[${PYTHON_USEDEP}]
+		dev-python/polib[${PYTHON_USEDEP}]
+		dev-python/pygments[${PYTHON_USEDEP}]
+		dev-python/qtpy[pyqt6,gui,network,${PYTHON_USEDEP}]
+		dev-python/send2trash[${PYTHON_USEDEP}]
```
Comment (bearsh) [inline dev-vcs/git-cola/git-cola-4.9.0.ebuild:25]: shouldn't `dev-python/send2trash` be remove here as you changed it to be a _optfeature_

Comment (stkw0) [reply dev-vcs/git-cola/git-cola-4.9.0.ebuild:25]: Yes, I missed that. Thanks!

**Comment by stkw0** [comment]
I did run pkgcheck scan. I seen the message about keywords being overshadowed and because I have always seen this message when adding a new unstable version when there is another unstable version on the tree I assumed this case where the same...

By chance there is some way to instruct pkgcheck to treat those cases differently? For example showing it in yellow if a package is added directly to stable.

**Comment by juippis** [comment]
> By chance there is some way to instruct pkgcheck to treat those cases differently? For example showing it in yellow if a package is added directly to stable.

It uses different colours based on severity of the issue. IIRC you can modify the level locally. By default, committing straight to stable is printed in red which is the colour for "ERROR".

**Pull request closed without merging**